content/rippled-api-methods/account_objects.md

```diff
@@ -63,7 +64,7 @@ The request includes the following parameters:
 | `Field`        | Type                                       | Description    |
 |:---------------|:-------------------------------------------|:---------------|
 | `account`      | String                                     | A unique identifier for the account, most commonly the account's address. |
-| `type`         | String                                     | _(Optional)_ If included, filter results to include only this type of ledger object. The valid types are: `offer`, `signer_list`, `state` (trust line), `escrow`, and `payment_channel`. <!-- Author's note: Omitted types from this list that can't be owned by an account: https://github.com/ripple/rippled/blob/1dbc5a57e6b0e90a9da0d6e56f2f5a99e6ac1d8c/src/ripple/rpc/impl/RPCHelpers.cpp#L676-L686 --> |
+| `type`         | String                                     | _(Optional)_ If included, filter results to include only this type of ledger object. The valid types are: `offer`, `signer_list`, `state` (trust line), `escrow`, and `payment_channel`. <!-- Author's note: Omitted types from this list that can't be owned by an account: https://github.com/ripple/rippled/blob/1dbc5a57e6b0e90a9da0d6e56f2f5a99e6ac1d8c/src/ripple/rpc/impl/RPCHelpers.cpp#L676-L686 --> <!-- Note 2: Can't filter by Checks yet. See https://github.com/ripple/rippled/issues/2350 for status. --> |
```

Comment:
Perchance add a TODO alongside your Author's notes?

nit: ordering the table to parallel the request JSON or alphabetically would add consistency.

Reply:
Also, reading this field description on its own makes me think checks are not valid types. I suggest modifying valid per the filtering, something like:
> If included, filter results to include only this type of ledger object. The valid types that can be filtered are: `offer`, `signer_list`, `state` (trust line), `escrow`, and `payment_channel`.

Also, might it be worth a note and link below about impending filterable checks?

Reply:
Updated this with the expectation that https://github.com/ripple/rippled/pull/2356 will get merged before 0.90.0 goes out.

Not adding "TODO" to the author's note because it's not an incomplete, it's a justification of why the docs don't match up 100% with the code.